src/mlpack/core/data/load.hpp

```diff
@@ -290,6 +290,51 @@ bool Load(const std::string& filename,
           format f = format::autodetect,
           std::enable_if_t<HasSerialize<T>::value>* = 0);
 
+/**
+ * This function a set of several dataset files into one matrix.
+ * This is usually the case if the dataset is collected on several occasions
+ * and not agglomerated into one file.
```

Comment:
```suggestion
 * and not agglomerated into one file, or if the dataset has been partitioned
 * into multiple files.
```
Up to you, I wanted to point out this common case too, but it's not a big deal either way.